openapi-v1.yaml

```diff
@@ -2772,27 +2810,65 @@ definitions:
       name: intro-to-genomics
       description: contains arrays and notebooks for an 101 course on genomics with TileDB
 
-  GroupListing:
+  GroupEntry:
+    description: An entry of a group, that is a subgroup or an asset
+    type: object
+    properties:
+      asset:
+        description: If not empty, this entry is an asset
+        $ref: "#/definitions/ArrayInfo"
+        x-omitempty: true
+      group:
+        description: If not empty, this entry is a group
+        $ref: "#/definitions/Group"
+        x-omitempty: true
+
+  GroupContents:
```

Comment:
This definition is only used in `public` endpoint. But I also think that public should return `GroupContents` like shared and owned (see next comment)